crates/update/src/main.rs

```diff
@@ -19,7 +19,13 @@ fn main() -> anyhow::Result<ExitCode> {
     if cmd == "spacetimedb-update" {
         spacetimedb_update_main()
     } else if cmd == "spacetime" {
-        proxy::run_cli(None, Some(argv0.as_os_str()), args.collect())
+        let args = args.collect::<Vec<_>>();
+        if args.first().is_some_and(|s| s == "version") {
+            // if the first arg is unambiguously `version`, go straight to `spacetime version`
+            spacetimedb_update_main()
```

Comment:
Q: is this case necessary for some reason? Won't the CLI end up delegating back to `spacetimedb-update` anyway?

Reply:
Yes, but there's no perfect way for the cli to find the update binary again. If you're running via cargo run or something, it'll find `cli_bin_path` in `~/.local`, but then `spacetime version` will delegate to a different binary. It all sucks, but I feel this avoids the most bad edge cases.